tests/vision/test_backbones.py

```diff
@@ -2,19 +2,14 @@
 from pytorch_lightning.utilities import _BOLTS_AVAILABLE, _TORCHVISION_AVAILABLE
 
 from flash.utils.imports import _TIMM_AVAILABLE
-from flash.vision.backbones import (
-    backbone_and_num_features,
-    bolts_backbone_and_num_features,
-    timm_backbone_and_num_features,
-    torchvision_backbone_and_num_features,
-)
+from flash.vision.backbones import IMAGE_CLASSIFIER_BACKBONES
 
 
 @pytest.mark.parametrize(["backbone", "expected_num_features"], [("resnet34", 512), ("mobilenet_v2", 1280),
                                                                  ("simclr-imagenet", 2048), ("vgg16", 512)])
 def test_backbone_and_num_features(backbone, expected_num_features):
```

Comment:
Merge all these tests together now that the models come from the same place?

It'd  also be nice that we assert the expected backbone model class

Reply:
Done !